### PR TITLE
mindset motivation overhaul

### PR DESCRIPTION
**Pull request by karenword**
First attempt at reorganizing mindset, motivation, and EIA content. Moves mindset in with motivation and pulls Accessibility, Stereotype Threat, and Imposter Syndrome out to be featured in a separate module. Trims mindset, adding rationale for inclusion and some motivation content. 
Main drawback is that EIA content is not ready to stand on its own. Next steps include:

- [x] consider what's necessary to support EIA content as a stand-alone episode and whether that's feasible in the scope of the current updates
- [ ] consider reference management. Do we want to continue to include direct links to referenced content, link to our References page (highlighting that resource), or create items for each reference in Zotero?
- [ ] review new timing, update estimates, and revise accordingly

### CODE REVIEW

**Comment by karenword** [comment]
@brownsarahm new (same) PR to the correct repo is here!

**Comment by karenword** [comment]
Discussed with @klbarnes20 who will apply some of her expertise to drafting revisions to the EIA section. Proposed list of changes is here:
- [ ] Modify curbcuts call-out to introduce UDL (Universal Design for Learning) -  Emphasize maximal preparation for all contingencies over responding to requests (but preserve link to host template re soliciting requests)
- [ ] Specific recommendations:
      -  Physical accessibility for in person workshops (Present)
      -  Assign a notetaker or captioner in person or online. Investigate if your institution has captioning available. Link to Daniel Chen’s blog post on technical setup for captioning. Maybe link to [command history instructions](https://github.com/4TUResearchData-Carpentries/documentation/blob/master/command-history.md)
     - note that our model lacks accessibility in ways that are beyond the control of Instructors (e.g. fully synchronous instruction). Change callout about PDFs to please innovate and share ways that you have made your workshop more accessible - this is how our model will improve
- [ ] Consider swapping out UK home office pdfs for a non-pdf UDL tips page? (maybe this one linked by readyset? Or something else created by an affected community?)
- [ ] Change stereotype threat to broader discussion of stereotypes in the classroom
     - Maybe including a note to be aware of inevitable bias
- [ ] Update imposter syndrome to flow better
     - Discuss instructor & learner experiences
     - Consider whether this section should stay with EIA or move back to motivation

File: _episodes/08-motivation.md [hunk 1]
```diff
@@ -9,76 +9,66 @@ questions:
...
 ![A stylized graph with y-axis labeled "usefulness once mastered" and and x-axis labeled "mean time to master". The upper left quadrant says "teach this first" and the lower right quadrant says "don't bother".](../fig/what-to-teach.png)
 
-Another way to think about the graph shown above is "authentic tasks."
-An authentic task is exactly what it sounds like -- a real task performed
+Another way to think about the graph shown above is **authentic tasks** -- real tasks performed
 by someone doing their work. If you can identify authentic tasks from your
 own work that could be useful to others, these examples will be highly
 motivating.
 
+
```
Comment (brownsarahm) [inline _episodes/08-motivation.md:65]: did this exercise with an additional intro that this was one of the more "meta" moments in training, where we're using the activity to both reinforce the material just covered *and* as another example of formative assessment.  both got great responses on the activity *and* follow up questions about how to use think, pair, share.

File: _episodes/08-motivation.md [hunk 2]
```diff
@@ -87,434 +77,271 @@ motivating.
...
-> ## Strategies for Motivating Learners
->
-> *[How Learning Works][worldcat-hlw]* by Susan Ambrose, et al.,
-> contains this list of evidence-based methods to motivate learners.
+> ## Brainstorming Motivational Impacts
 >
-> In groups of two or three, pick three of these points and briefly describe
-> in the Etherpad how we can apply these strategies in our workshops.
+> Think back to courses you have taken in the past and consider things that an instructor has said or done that you found either **motivating** or **demotivating**. 
+> Try to think of one example in each case, and share your example under the appropriate heading in the etherpad.
```
Comment (brownsarahm) [inline _episodes/08-motivation.md:107]: did this exercise as described in the etherpad.

File: _episodes/08-motivation.md [hunk 3]
```diff
@@ -87,434 +77,271 @@ motivating.
...
+often the 'first responders' to learner mistakes. 
+
+
+> ## Helping Learners Learn From Mistakes (optional)
+> A learner at your workshop asks for your help with an exercise and shows you their attempt at solving it. You see they've made an error
+> that shows they misunderstand something fundamental about the lesson (for example, in the shell lesson, they forgot to put a space between
+> `ls` and the name of the directory they are looking at). What would you say to the learner?
 >
-> If time, what themes do you see among the stories?  What are positive actions you could take to avoid these?
+> In the Etherpad, describe the error your learner has made and how you would respond.
```
Comment (brownsarahm) [inline _episodes/08-motivation.md:153]: did this, but with mixed success of trainees describing a mistake in their own lesson vs describing what they'd do for the missing space.  

Maybe explicitly recommend choosing an exercise in the lesson they've been teaching from?

File: _episodes/08-motivation.md [hunk 4]
```diff
@@ -87,434 +77,271 @@ motivating.
...
+What we have not discussed yet is strategies to motivate the *instructor*. But why does *your* motivation matter?
++ **Learners respond to an instructor's enthusiasm**. The more motivated you are, the more motivated they will be!
++ **Instructors are learning to teach.** This takes motivation, too! Deliberative practice, seeking feedback, and reflecting on mistakes in the
+context of your own busy work life is a challenge. What will keep you energized to stay engaged with your learning process?
++ **Carpentries Instructors teach because they want to.** Whether you are truly volunteering your time or are fulfilling a role in a job you have 
+chosen, teaching is something you came here motivated to do. Teaching can be an incredibly gratifying activity! Finding and preserving your own motivation
+through the many challenges ahead will make your journey as a teacher a more joyful one. 
 
-> ## Learning about Accessibility
+> ## Why Do You Teach? (Optional)
```
Comment (brownsarahm) [inline _episodes/08-motivation.md:246]: did this exercise

File: _episodes/09-eia.md [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+> ## What Happens When Accessibility is an Issue?
+>
+> Think of a time when you've been affected by, or noticed someone else
+> being affected by issues with accessibility. This may have been at a
+> conference you attended where the elevator was out of service, or maybe
+> a class you were taking relied on audio delivery of content. Describe what happened,
+> how it impacted your (or someone else's) ability to be involved and what could
+> have been done to provide better accessibility in this case.
+>
+> This exercise should take about 5 minutes.
```
Comment (brownsarahm) [inline _episodes/09-eia.md:43]: skipped this exericise

File: _episodes/09-eia.md [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+> The [UK Home Office](https://hodigital.blog.gov.uk/category/accessibility/) has put together a set of posters of dos and don'ts for making
+> visual and web-based materials more accessible for different populations. Take a look at one of
+> [these posters](https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf) and
+> put one thing you've learned in the Etherpad.
+>
+> Note: There are [translations](https://github.com/UKHomeOffice/posters/tree/master/accessibility)
+> available in a number of languages, including Dutch, French, Spanish, Swedish, Portuguese, German, and
+> Chinese.
+>
+> This exercise should take about 5 minutes.
```
Comment (brownsarahm) [inline _episodes/09-eia.md:87]: gave some extra time for this one, it never finishes in 5 minutes.  It also fostered some great discussion and appreciation

File: _episodes/09-eia.md [hunk 7]
```diff
@@ -0,0 +1,336 @@
...
+{: .challenge}
+
+> ## Accessibility and PDFs
+>
+> The posters above are in PDF format - which itself can sometimes be inaccessible to people
+> who use screen readers. If you know of a similar resource describing accessibility guidelines
+> that would be more accessible than what's linked above, open an issue on our instructor
+> training curriculum respoistory here: https://github.com/carpentries/instructor-training/issues
+{: .callout}
```
Comment (brownsarahm) [inline _episodes/09-eia.md:96]: missed this :-/

File: _episodes/09-eia.md [hunk 8]
```diff
@@ -0,0 +1,336 @@
...
+For example,
+sites like [WebAIM][webaim] allow you to check
+how accessible your online materials are to users with no or limited vision.
+
+The third,
+and most important,
+is to *involve people with disabilities in decision-making*.
+Our mailing lists are a good place to ask for advice,
+and updates to [our accessibility checklist](https://docs.carpentries.org/topic_folders/hosts_instructors/workshop_needs.html#accessibility)
+are always welcome.
```
Comment (brownsarahm) [inline _episodes/09-eia.md:116]: I'm not sure I said these points, the conversation was a little unstructured for a little, but I do like this.

File: _episodes/09-eia.md [hunk 9]
```diff
@@ -0,0 +1,336 @@
...
+{: .challenge}
+
+
+While it may not be possible to accommodate everyone's needs,
+it *is* possible to get a good working structure in place
+without any knowledge of what specific disabilities people might have.
+Having at least some accommodations prepared also makes it clear that
+hosts and instructors care enough to have thought about problems in advance,
+and that any additional concerns are likely to be addressed.
+
```
Comment (brownsarahm) [inline _episodes/09-eia.md:53]: this point was greatly appreciated, esp when paired with following exercise

File: _episodes/09-eia.md [hunk 10]
```diff
@@ -0,0 +1,336 @@
...
+> ## Overcoming Imposter Syndrome (Optional)
+>
+> Think of a time when learning something was difficult for you, or
+> you made a mistake that seemed silly or embarrassing. Is that task still hard for you?
+> In the Etherpad, describe how you might use this as a motivational story to help your learners
+> overcome their own imposter syndrome.
+>
+> This exercise should take about 5 minutes.
+{: .challenge}
```
Comment (brownsarahm) [inline _episodes/09-eia.md:256]: skipped this

**Review by brownsarahm — COMMENTED** [review]
Overall, I like this revision a lot. This "review" includes inline comments based on piloting teaching from this. 

An idea that came up after the training meeting where people complained about the current discussion of stereotype threat, etc:  We point out that our goal as intructors is a minimal, working mental model and we eventually give trainees the point that a usable working model is often incomplete and sometimes even wrong, in simplifying it for people to understand.  Maybe we could rely on that idea for how we're covering these points.  That, we are aiming to give them some actionable advice to create an inclusive learning environment, but that we're not able to cover these ideas in full.  We're hoping that they have a *useful* mental model of these ideas, but that it's necessarily incomplete and slightly oversimplified. Then note that there are links to literature.  

If we go that route, we can introduce and foreshadow that that will happen when we talk about mental models originally and then come back to it here.  When we bring that point back, we can also return to the "do no harm" idea and note that oversimplifying things too far increases risk of causing harm?  

This is definitely not a fully formed idea, but sharing anyway with hope that you two can use it maybe?

File: _episodes/08-motivation.md [hunk 11]
```diff
@@ -87,434 +77,271 @@ motivating.
...
-> editing a text file seems like a simple task,
+> editing a text file seems like a quick task,
 > but most graphical editors save things to the user's desktop or home directory.
-> If people need to run shell commands on the files they've edited,
-> a substantial fraction won't be able to navigate to the right directory without help.
-> If this seems like a small problem to you,
-> please revisit the discussion of [expert awareness gap]({{ page.root }}{% link _episodes/03-expertise.md %}#expertise-and-teaching).
+> If a novice needs to run shell commands on the files they've edited,
+> they often fail to navigate to the right directory without help.
+> You will learn to anticipate these sorts of hang-ups as you chart your [expert awareness gaps]({{ page.root }}{% link _episodes/03-expertise.md %}#expertise-and-teaching). As a result, your skill at estimating time to mastery will improve.
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:88]: "hang-ups" --> "challenges" to avoid regional idioms

File: _episodes/08-motivation.md [hunk 12]
```diff
@@ -87,434 +77,271 @@ motivating.
...
+> If you are new to teaching, try to ask an experienced instructor for feedback before trying out a new exercise.
 {: .callout}
 
-## Other Motivational Strategies
+
+While we aim to begin workshops with motivating content, in practice this does not always occur. Workflow-based content like that taught in Data Carpentry
+workshops may start at the beginning of the workflow, for example. Even when a 'motivating example' is built in to the start of a workshop, technical 
+problems like software installation can turn those precious first minutes into an experience of frustration and impatience. That is ok! What is important is to
+**be mindful of times when your content is not motivating**, and to strategize your rescue using some of the other techniques in this section.
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:96]: YES!!

File: _episodes/08-motivation.md (hunk 2 above)
Comment (ErinBecker) [inline _episodes/08-motivation.md:107]: Pedantic reminder to capitalize Etherpad. (@gvwilson said this to me once and has successful made me hyper-conscious of this! 😆 )

Comment (ErinBecker) [reply _episodes/08-motivation.md:107]: I like the idea of combining the motivating and demotivating examples into a single exercise. Nice!

Comment (ErinBecker) [reply _episodes/08-motivation.md:107]: Can we also prompt trainees to identify something that could have been done to make that situation not demotivating (solutions oriented)?

File: _episodes/08-motivation.md [hunk 13]
```diff
@@ -87,434 +77,271 @@ motivating.
...
-> *[How Learning Works][worldcat-hlw]* by Susan Ambrose, et al.,
-> contains this list of evidence-based methods to motivate learners.
+> ## Brainstorming Motivational Impacts
 >
-> In groups of two or three, pick three of these points and briefly describe
-> in the Etherpad how we can apply these strategies in our workshops.
+> Think back to courses you have taken in the past and consider things that an instructor has said or done that you found either **motivating** or **demotivating**. 
+> Try to think of one example in each case, and share your example under the appropriate heading in the etherpad. 
 >
-> * Strategies to Establish Value
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:116]: I approve of cutting this list!

File: _episodes/08-motivation.md [hunk 14]
```diff
@@ -87,434 +77,271 @@ motivating.
...
-> *Think* back to a computational (or other) course you took in the past, and
-> identify one thing the instructor did that motivated you.
-> *Pair* up with your neighbor and discuss what motivated you.
-> *Share* the motivational story in the Etherpad.
->
-> This exercise should take about 5 minutes.
-{: .challenge}
+Motivation is supported by active engagement. Participation allows learners to ask questions, resolving roadblocks quickly, and demonstrate knowledge, building 
+confidence. It also facilitates learning! However, in a room full of strangers, **most learners will not immediately feel comfortable speaking up**, especially when they feel 
+confusion or doubt. Creating a motivating classroom means inviting communication and reinforcing that invitation with an attentive response.
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:116]: Love this

File: _episodes/08-motivation.md [hunk 15]
```diff
@@ -87,434 +77,271 @@ motivating.
...
-> This exercise should take about 5 minutes.
-{: .challenge}
++ **Establishing norms for interaction**. This can be done by creating procedures for communication, e.g. turn taking in discussions, passing around a 'talking 
+stick', or encouraging quieter people to contribute. Having, discussing, and enforcing a [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html) 
+also provides a framework for positive communication to occur. 
++ **Encouraging learners to learn from each other**. Working in pairs, or "[pair programming](https://en.wikipedia.org/wiki/Pair_programming)," encourages 
+learners to talk through their learning process, reinforcing memory and making it more likely that confusion will be expressed and 
+resolved. This can also address challenges of varying background experience: asking more advanced learners to help beginners can 
+maximize learning for both. In these cases, make sure the beginner is doing the typing!
++ **Acknowledging when learners are confused**. Acknowledging and exploring confusion with kindness rewards learners for sharing vulnerable information. It also
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:127]: I love this so much

File: _episodes/08-motivation.md [hunk 16]
```diff
@@ -87,434 +77,271 @@ motivating.
...
+> {: .solution}
+{: .challenge}
+
++ **Leveraging the Power of "Yet"**. A request for help might start with "I can't ___ " or "I don't understand ___ ". Depending on the attitude of the learner, 
+these can sound like statements of fact rather than requests for help! Adding the word "yet" to the end of these sentences helps emphasize that being a novice is 
+a temporary state, and encourages a growth mindset towards progress. 
+
+### First, Do No Harm!
+
+When learning a skill, we develop more than expert awareness gaps -- we also develop Opinions about tools and methods, and sometimes base a professional identity
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:201]: is this capital O "Opinions" on purpose?

File: _episodes/08-motivation.md [hunk 17]
```diff
@@ -87,434 +77,271 @@ motivating.
...
+{: .challenge}
+
++ **Leveraging the Power of "Yet"**. A request for help might start with "I can't ___ " or "I don't understand ___ ". Depending on the attitude of the learner, 
+these can sound like statements of fact rather than requests for help! Adding the word "yet" to the end of these sentences helps emphasize that being a novice is 
+a temporary state, and encourages a growth mindset towards progress. 
+
+### First, Do No Harm!
+
+When learning a skill, we develop more than expert awareness gaps -- we also develop Opinions about tools and methods, and sometimes base a professional identity 
+around displaying technical expertise. Technical boasts, digs, and other showy moves can score points in conversation with fellow experts, but these present
```
Comment (ErinBecker) [inline _episodes/08-motivation.md:202]: "digs" --> "insults" to avoid regional idioms

**Comment by karenword** [comment]
Addresses https://github.com/carpentries/instructor-training/issues/845

**Comment by karenword** [comment]
Addresses https://github.com/carpentries/instructor-training/issues/618 to the extent that it re-frames how and why we discuss mindset, which may make the broader theoretical controversies more clearly out of scope.

**Comment by karenword** [comment]
Note: this PR was merged before it was fully finished in order to allow additional changes to be clearly evident, particularly for the new EIA file.

**Pull request merged**